### DISCUSSION TITLE
Tags shown on pages don't automatically update when modified (and saved) in the administration panel

### DISCUSSION THREAD

**Discussion by RiskeyL** [requarks/wiki#6505] [Error / Bug Report]
## Description
I modified a tag and saved in the adminstration panel, and then open a page that was marked with that tag and discovered that the tag shown on the page didn't update to the modified version, even after a page refreshing. Then, I entered the editor mode of that page, and found in the page matadata that the tag was already modified. Then, I quit the editor mode, refreshed the page, and the tag still didn't update.

## Reproduction steps
1. Tag Page  A with "test".
2. In the administration panel, modify the tag and label of "test" to "test1", and click **Save**.
3. Open Page A. The tag shown is still "test".

BTW, what is the difference between **Tag** and **Label** of a tag?
![image](https://github.com/requarks/wiki/assets/36894937/3da2cdbd-41fb-4e09-a9b2-740f2d870323)

**Comment by RiskeyL** [comment]
Any feedback?? T^T

**Comment by RiskeyL** [comment]
Any feedback??? Please help.

**Comment by RiskeyL** [comment]
Asking for help

**Comment by riversdev0** [comment]
This issue is affecting me, also.  I looked in the database and noticed that the data is updated properly.  For me, restarting the Wiki.js docker container made the new tag name start appearing correctly.  There must be some type of cache that Wiki.js maintains that isn't purged when a tag is renamed.

**Comment by riversdev0** [comment]
I kept digging and found that tags will appear correctly after you flush the cache from Admin > Utilities > Flush Cache > Flush Pages and Assets Cache.
